tests/test_routes.py

```diff
@@ -0,0 +1,181 @@
+from operator import contains
+from werkzeug.exceptions import HTTPException
+from app.models.board import Board
+from app.models.card import Card
+import pytest
+
+
+BOARD_TITLE = "Favorite Quotes"
+BOARD_OWNER = "Talia"
```

Comment:
These variables could be imported from the conftest file (or a separate test data file) so they do not need to be duplicated.